components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4296,6 +4296,36 @@ def searchProcessing():
             search.close()
         return rv
 
+    def getThumbnailSet(self, image_ids, max_size=64):
```

Comment:
so how about if we deprecate `getThumbnail(self, size=(64, 64)...)` in fav of `getThumbnail2(self, size=(96, 96)...)` and add warning about size change and explain it there ?